index.html

```diff
@@ -77,22 +78,37 @@ <h3>Load a URL</h3>
   <h3>Load a Source</h3>
   <select id=load-source>
     <optgroup label="hls">
-      <option value="https://d2zihajmogu5jn.cloudfront.net/bipbop-advanced/bipbop_16x9_variant.m3u8">Muxed TS with 1 alt Audio, 5 captions</option>
+      <option value="https://d2zihajmogu5jn.cloudfront.net/bipbop-advanced/bipbop_16x9_variant.m3u8">Bipbop - Muxed TS with 1 alt Audio, 5 captions</option>
```

Comment:
What do you think of loading all of these sources from a json file?

Reply:
Not a bad idea, I can do it as part of another PR, would like to get this in so it's easier for people to test.